clang/test/CIR/CodeGen/tbaa.c

```diff
@@ -0,0 +1,121 @@
+// modified from clang/test/CodeGen/tbaa.c
+// with clangir enabled and some debugging info
+// if disable debuggging info is supported by clangir
+// the debugging info can be removed
+// and keep the test case the same with clang/test/CodeGen/tbaa.c
```

Comment:
Nit, let's make this comment a bit more terse (for this and other test files), example:
```
// This is inspired from <FILE>, with both CIR and LLVM checks.
// FIXME: use identical LLVM checks once
// https://github.com/llvm/clangir/issues/793 is fixed.
```